packages/client/src/styles/editorial.css

```
@@ -43,6 +43,50 @@
+  width: 42%;
+  background: rgb(var(--text-strong-950) / 0.08);
+  content: "";
+  filter: blur(var(--blur-material-regular));
+  transform: translateX(0);
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .editorial-skeleton::after {
+    animation: editorial-skeleton-sweep
+      calc(
+        var(--spring-effects-slow-duration) + var(--spring-effects-slow-duration) +
```

Stylelint (17.14.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)